mmv1/products/gemini/RepositoryGroup.yaml

```diff
@@ -0,0 +1,147 @@
+examples:
+  - name: "gemini_repository_group_basic"
+    min_version: 'beta'
+    primary_resource_id: "example"
+    primary_resource_name: 'acctest.BootstrapSharedCodeRepositoryIndex(t, "basic-rg-example%{random_suffix}", "us-central1", "", ""), "test-rg-repository-group-id"'
+    vars:
+      repository_group_id: "example-repository-group-id"
+      git_repository_link_id: 'example-git-repository-link-id'
+      cri_id: "cri-example"
+      repository_resource: "projects/example-project/locations/us-central1/connections/example-connection/gitRepositoryLinks/example-repo"
+      connection_id: "example-connection-id"
+    test_vars_overrides:
+      repository_group_id: '"example_repository_group_idx"'
+      git_repository_link_id: 'acctest.BootstrapGitRepository(t, "basic", "us-central1", "https://github.com/CC-R-github-robot/tf-test.git", acctest.BootstrapDeveloperConnection(t, "basic", "us-central1", "projects/502367051001/secrets/tf-test-cloudaicompanion-github-oauthtoken-c42e5c/versions/1", 54180648))'
+      cri_id: 'acctest.BootstrapSharedCodeRepositoryIndex(t, "test-rg", "us-central1", "", "")'
+      repository_resource: '"projects/"+envvar.GetTestProjectFromEnv()+"/locations/us-central1/connections/"+acctest.BootstrapDeveloperConnection(t, "basic", "us-central1", "projects/502367051001/secrets/tf-test-cloudaicompanion-github-oauthtoken-c42e5c/versions/1", 54180648)+"/gitRepositoryLinks/"+acctest.BootstrapGitRepository(t, "basic", "us-central1", "https://github.com/CC-R-github-robot/tf-test.git", acctest.BootstrapDeveloperConnection(t, "basic", "us-central1", "projects/502367051001/secrets/tf-test-cloudaicompanion-github-oauthtoken-c42e5c/versions/1", 54180648))'
+      connection_id: 'acctest.BootstrapDeveloperConnection(t, "basic", "us-central1", "projects/502367051001/secrets/tf-test-cloudaicompanion-github-oauthtoken-c42e5c/versions/1", 54180648)'
+    exclude_test: true
+timeouts:
+  insert_minutes: 30
+  update_minutes: 30
+  delete_minutes: 30
+autogen_async: true
+async:
+  operation:
+    timeouts:
+      insert_minutes: 30
+      update_minutes: 30
+      delete_minutes: 30
+    path: name
+    base_url: "{{op_id}}"
+    wait_ms: 1000
+  actions:
+    - create
+    - delete
+    - update
+  type: OpAsync
+  result:
+    resource_inside_response: true
+    path: response
+  error:
+    path: error
+    message: message
+  include_project: false
+iam_policy:
+  min_version: 'beta'
+  parent_resource_attribute: 'repository_group_id'
+  method_name_separator: ':'
+  fetch_iam_policy_verb: 'GET'
+  set_iam_policy_verb: 'POST'
+  import_format:
+    - 'projects/{{project}}/locations/{{location}}/codeRepositoryIndexes/{{code_repository_index_id}}/repositoryGroups/{{repository_group_id}}'
+    - '{{repository_group_id}}'
+  allowed_iam_role: 'roles/cloudaicompanion.repositoryGroupsUser'
+error_retry_predicates:
+  - 'transport_tpg.IsRepositoryGroupQueueError'
```

Comment:
guess, you would like to add the following isCodeRepositoryIndexUnreadyError here